lab/src/Microsoft.ServiceModel.Syndication/src/ServiceDocument.cs

```diff
@@ -70,16 +69,16 @@ public Collection<Workspace> Workspaces
             }
         }
 
-        public static ServiceDocument Load(XmlReader reader)
+        public static Task<ServiceDocument> Load(XmlReader reader)
         {
```

Comment:
What's the purpose of this method? It returns Task<ServiceDocument> so should be called LoadAsync, but then that's the same as the method below. I think you can just remove this method as it doesn't provide anything beyond specifying the generic type to LoadAsync which a customer can do.